### DISCUSSION TITLE
Releases atom feed on private repo behind SAML SSO

### DISCUSSION THREAD

**Discussion by mskrip** [community/community#27888] [Apps, API and Webhooks]
It seems my unanswered question was [lost during migration to the new GH community](https://github.com/orgs/community/discussions/27346#discussioncomment-3275305) without me getting any notification the question was basically deleted. So I'm posting it again here.

The gist of the issue is that requesting `releases.atom` on a private repository which is owned by an organization with SAML SSO is not possible.

- The request just downloads a HTML page to authorize the request with SSO
- The PAT used for the request is authorized to access the organization and should have correct permissions to access the requested resource

Previously handled by @liligalante (@queenofcorgis)

Link to the original question: https://github.community/t/releases-atom-feed-on-private-repo-behind-saml-sso/233397

**Comment by mskrip** [comment]
@queenofcorgis Any news on this?

**Reply by queenofcorgis** [reply]
@mskrip thanks for following up and being so patient - I'm looking into documentation on this!

**Reply by mskrip** [reply]
@queenofcorgis This behavior still persists

**Reply by queenofcorgis** [reply]
There is an internal Issue with folks at GitHub for this and I have pinged them for updates, but nothing yet, once there is movement you'll be the first to know!

**Reply by mskrip** [reply]
Maybe as an additional piece of information to add here: using the new fine-grained personal access token does not fix the issue either, since that token is valid only for authenticating against the REST API, while my issue was with accessing the `github.com/<owner>/<repo>/releases.atom` which is not a REST API endpoint but the classic PAT works with it (at least of normal private repositories).

**Comment by arqu-csl** [comment]
@queenofcorgis @mskrip  was this ever addressed / solved?  we are experiencing the same issue where the `releases.atom` is returning "Not Found".  Users are authenticated and can otherwise view the `/releases` page.  (via the GH web UI, not API)
We are similar in that we have SAML SSO for our enterprise cloud account as well.

Thanks.

**Reply by mskrip** [reply]
Not solved yet for me. `releases.atom` accessed through the UI returns "Not Found" on all private repositories as far as I'm aware. But on private repositories you can access the page through API with appropriate PAT. However, for repositories owned by organization with SAML SSO the `releases.atom` is not properly accessible even with the PAT.

**Reply by benjamin-mogensen** [reply]
Same issue here - was trying to get an Electron app to be auto updated from our organisations private repo, but had to give up as it relies on the releases.atom page to be accessible.

**Comment by benny-yamagata** [comment]
Is this still an issue?  I am trying to access this from a script and I get `Not Found`, even if I just try to access the feed myself in the browser by appending `.atom` it still gives me back `Not Found`.

Kinda crazy that there is no documentation around this.
